test/e2e/shared/template.go

```diff
@@ -120,6 +120,9 @@ func newBootstrapTemplate(e2eCtx *E2EContext) *cfn_bootstrap.Template {
 	t.Spec.EKS.DefaultControlPlaneRole.Disable = false
 	t.Spec.EKS.ManagedMachinePool.Disable = false
 	t.Spec.S3Buckets.Enable = true
+	t.Spec.IPv6 = &bootstrapv1.IPv6{
```

Comment:
As we are enabling it by default in the e2e tests then perhaps we should enabling it by default in capa?